inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -277,10 +277,11 @@ void MKLDNNGraphOptimizer::FuseDeconvolutionAndSimpleOperation(MKLDNNGraph &grap
         }
 
         auto childNode = parentNode->getChildEdgeAt(0)->getChild();
-        // at this moment deconvolution supports only depthwise as post op
-        if (!childNode->canBePerformedAsScaleShift(parentNode.get())) {
-            parent++;
-            continue;
+        if (!parentNode->canFuse(childNode)) {
+            if (!(parentNode->getFusedWith().empty() && childNode->canBePerformedAsScaleShift(parentNode.get()))) {
+                parent++;
+                continue;
+            }
```

Comment:
Is not it better to move this code into the `canFuse` function?

Reply:
Thanks, I thought about it, but I cannot call this method `childNode->canBePerformedAsScaleShift(parentNode.get())` from Deconvolution node, so I decided to write it like this.